pkg/apis/devops/v1alpha2/register.go

```diff
@@ -216,18 +216,18 @@ func addWebService(c *restful.Container) error {
 		Returns(http.StatusOK, RespOK, []devops.JenkinsCredential{}).
 		Reads([]devops.JenkinsCredential{}))
 
-	// match Jenkisn api "/blue/rest/organizations/jenkins/pipelines/{projectName}/{pipelineName}"
+	// match Jenkisn api "/blue/rest/organizations/jenkins/pipelines/{devops}/{pipelineName}"
 	webservice.Route(webservice.GET("/devops/{devops}/pipelines/{pipelines}").
 		To(devopsapi.GetPipeline).
 		Metadata(restfulspec.KeyOpenAPITags, tags).
 		Doc("Get a Pipeline Inside a DevOps Project").
 		Param(webservice.PathParameter("pipelines", "the name of pipeline, which helps to deliver continuous integration continuous deployment.")).
```

Comment:
the path should be pipelineName, right?

Reply:
here should be pipeline, and all api schema is "${}s/${}".

Reply:
fix comment